backend/src/main/java/feedzupzup/backend/feedback/domain/Category.java

```diff
@@ -0,0 +1,30 @@
+package feedzupzup.backend.feedback.domain;
+
+import feedzupzup.backend.global.exception.ResourceException.ResourceNotFoundException;
+import java.util.Arrays;
+
+public enum Category {
+
+    FACILITY("시설"),
+    ADMINISTRATION("행정"),
+    CURRICULUM("커리큘럼"),
+    ETC("기타"),
+    ;
+
+    private final String koreaName;
+
+    Category(final String koreaName) {
+        this.koreaName = koreaName;
+    }
+
+    public static Category from(String value) {
+        return Arrays.stream(Category.values())
+                .filter(category -> category.koreaName.equals(value))
+                .findFirst()
+                .orElseThrow(() -> new ResourceNotFoundException("존재하지 않는 카테고리입니다."));
+    }
```

Comment:
p2)
from보다는 찾고, 없으면 예외를 발생시키는 것과 같다고 생각해서
아래 메서드 명은 어떤가요?
```suggestion
    public static Category findByKoreaName(String koreaName) {
        return Arrays.stream(Category.values())
                .filter(category -> category.koreaName.equals(value))
                .findFirst()
                .orElseThrow(() -> new ResourceNotFoundException("존재하지 않는 카테고리입니다."));
    }
```

Reply:
good. 반영할게요